bundles/org.openhab.automation.jythonscripting/README.md

```diff
@@ -1,13 +1,6 @@
-# Jython Scripting (DEPRECATED)
+# Jython Scripting
 
-::: tip Note:
-Currently, the development of Jython stopped at version 2.7 with no definite timeline to support Python 3.x.
-The 3rd party openHAB helper library for Jython is also no longer maintained.
-We would not recommend using Jython scripting at this point in time.
```

Comment:
Can you elaborate how this has changed?

Reply:
The python 2 specification and cpython is deprecated, but the jython implementation is not deprecated.

https://github.com/jython/jython/issues/312

that means the developer will continue improving and fixing jython until they have a successor.

I followed the discussions in the jython communitiy during the past weeks and I can confirm that they still fixing and improving it. Including support of newer java versions.

Regarding the helper libaries, I thing this should not be a stopper, because a lot of people, including me can live without these. We just want to use pythonin openhab. 

HabApp is in some cases not an options e.g. if you want to use actions.

Reply:
Okay, so according to this we should remove "the development of Jython stopped at version 2.7", and also "(DEPRECATED)" in the title. But I guess it's still true that the helper library is no longer maintained (of course this could be said about a lot of bindings, but automations seems much more crucial as such a central component being heavily depended upon). And perhaps it would still be worth providing a slight general warning for new adopters, or WDYT?

Reply:
The way the Jython addon has always been was that there is no official helper library from the start. This is the exact same situation as Groovy.

We could say "jsscripting and jruby has a mature helper library, you should try it" but then again:
a. If you want Python, you want Python. You're not going to look at other languages
b. If you aren't sure, you'll soon see how spartan the README page is compared to jsscripting and jruby's. This is also the same as Groovy.

Perhaps knocking off Python from the list of featured / recommended automation addon is sufficient.